packages/sdk/src/index.ts

```diff
@@ -1,14 +1,47 @@
+import type { AppRouter } from '@orderbook/routes'
 import { hc } from 'hono/client'
-import { http, type Account, type HttpTransport, type PublicClient, createPublicClient } from 'viem'
+import {
+	http,
+	type Address,
+	type HttpTransport,
+	type PublicClient,
+	type SignMessageParameters,
+	type SignMessageReturnType,
+	type SignTypedDataParameters,
+	type SignTypedDataReturnType,
+	type SignableMessage,
+	createPublicClient,
+} from 'viem'
 import { base } from 'viem/chains'
-import type { AppRouter } from '../../orderbook/src/routes/index'
+
+// Обобщенный тип для параметров подписи
+type GenericSignMessageParameters = {
+	message: SignableMessage
+	account?: Address
+}
+
+// Обобщенный тип для параметров подписи типизированных данных
+type GenericSignTypedDataParameters = {
+	domain: Record<string, any>
+	types: Record<string, Array<{ name: string; type: string }>>
+	primaryType: string
+	message: Record<string, any>
+	account?: Address
+}
+
+// Обновленный тип SDKAccount
```

Comment:
Can we convert these to English?

Reply:
my bad :) i removed these generated comments